src/app/client/src/app/modules/manage/components/user-org-management/user-org-management.component.ts

```diff
@@ -160,7 +160,7 @@ export class UserOrgManagementComponent implements OnInit, AfterViewInit {
   }
 
   public getUserJSON() {
-    this.manageService.getData(this.slug, this.userJSON).subscribe(
+    this.manageService.getData(this.userJSON, `${this.slug}.json`).subscribe(
```

Comment:
Why don't you create the map & use it instead of so many variables & functions defined for this?

Reply:
Yes, I agree.  This component needs refactoring because of duplicate code. Due to time constraints, I made the changes in the existing code itself.